README.md

```diff
@@ -1,10 +1,10 @@
-<img src="https://github.com/thrasher-/gocryptotrader/blob/master/web/src/assets/page-logo.png?raw=true" width="350px" height="350px" hspace="70">
+<img src="https://github.com/thrasher-corp/gocryptotrader/blob/master/web/src/assets/page-logo.png?raw=true" width="350px" height="350px" hspace="70">
 
 [![Build Status](https://travis-ci.com/thrasher-/gocryptotrader.svg?branch=master)](https://travis-ci.com/thrasher-/gocryptotrader)
```

Comment:
This and most of the other readme's are still pointing to the old link [Travis](https://travis-ci.com/thrasher-corp/gocryptotrader) has updated

codecov is also he same in a few of the sub package readmes
Don't want to tag them all since there is 59

Unsure if intended or not

Reply:
Thanks! Have just pushed an update to correct those